spec/fixtures/vcr_cassettes/api/v1/build_distributions/subset/timeout.yml

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
I believe no cassettes at all should be recorded when the domain is fake. You could kill your http server running on 3000 port before you run tests.